.github/workflows/docker_build_images.yml

```diff
@@ -77,9 +157,11 @@ jobs:
 
   build_and_push_components:
     name: Build and push vitess components Docker images
```

Comment:
```suggestion
    name: Build and push
```

Reply:
The actual component name is added so that this becomes something like "Build and push (bookworm, vtadmin)"

Reply:
It is good to keep the final display string short so that we can see CI status at a glance. Otherwise it gets truncated in the web UI and you have to click through to figure out exactly which workflow failed.

Reply:
That's a good point, in the screenshot it looks a bit overloaded.